src/app/preferences/components/Routes/Routes.tsx

```diff
@@ -18,6 +21,7 @@ import { getRelativeRoute } from "@/app/utils";
 
 const Routes = (): JSX.Element => {
   const base = urls.preferences.index;
+  const { sidePanelContent } = useSidePanel();
```

Comment:
Delete SSHKey sidePanel should be setup in a way that's consistent with API key delete side panel. It's activated via own route `/r/account/prefs/api-keys/{id}/delete`.

```suggestion
```

Reply:
This would be problematic because the ssh key deletion takes an array of values for multiple deleting, when the table is grouped, having it as a URL route will only account for a single ID

Reply:
Passing multiple ids is perfectly valid as well via query parameters.

e.g. `/r/account/prefs/ssh-keys/delete?ids=1,2`